### PR TITLE
ENT-12140: Update dependency tables

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

**Review by cf-bottom — CHANGES_REQUESTED** [review]
Please use a company e-mail instead of c***@unreasonablefarm.org

File: README.md [hunk 1]
```diff
@@ -125,37 +125,37 @@ File `install-dependencies` and the relevant subdirectories in `deps-packaging`
...
 | [libacl](http://download.savannah.gnu.org/releases/acl/)                         | 2.3.2  | 2.3.2  | 2.3.2  |                          |
 | [libattr](http://download.savannah.gnu.org/releases/attr/)                       | 2.5.2  | 2.5.2  | 2.5.2  |                          |
-| [libcurl](http://curl.haxx.se/download.html)                                     | 8.7.1  | 8.7.1  | 8.8.0  |                          |
+| [libcurl](http://curl.haxx.se/download.html)                                     | 8.9.1  | 8.8.0  | 8.8.0  |                          |
 | [libgnurx](http://www.gnu.org/software/rx/rx.html)                               | 2.5.1  | 2.5.1  | 2.5.1  | Windows Enterprise agent |
 | [libiconv](http://ftp.gnu.org/gnu/libiconv/)                                     | 1.17   | 1.17   | 1.17   | Needed by libxml2        |
-| [libxml2](http://xmlsoft.org/sources/)                                           | 2.12.6 | 2.12.6 | 2.13.1 |                          |
+| [libxml2](http://xmlsoft.org/sources/)                                           | 2.13.3 | 2.13.1 | 2.13.1 |                          |
```
Comment (olehermanse) [inline README.md:136]: Curious to see 3.21.x on newer version than master and 3.24.x, but I see there was a PR merged for 3.21, and still pending for other branches, so probably makes sense.

**Review by olehermanse — APPROVED** [review]
Thanks for working on this :)

File: README.md (hunk 1 above)
Comment (craigcomstock) [reply README.md:136]: Correct. It is a bit odd but the master and 3.24.x deps updates are works in progress as 3.21 was the immediate need. Thanks!

**Pull request merged**